examples/image_classifier/streaming_video_client_side_batching/custom_handler.py

```diff
@@ -0,0 +1,48 @@
+import base64
```

Comment:
Would suggest adding a simple test here https://github.com/pytorch/serve/tree/master/test/pytest

Reply:
added a pytest but this is being ignored for the regression tests. So, not sure if its really useful